### ISSUE TITLE
Release Gammapy 0.7

### ISSUE THREAD

**Issue by cdeil** [gammapy/gammapy#1165]
I would like to cut a Gammapy 0.7 release soon. I have some time this week, but I'm also happy to wait a bit if needed. Please note that I'd be happy to do releases more frequently from now on, if something doesn't get into this release, there can be an 0.8 a few weeks later.

The 0.6 release is from April 2017 and there have been a lot of fixes and additions:
http://docs.gammapy.org/en/latest/changelog.html

@woodmd @adonath @joleroi @jjlk @bkhelifi - Do you have something that you want in the 0.7 release that I we should wait for?

All - if possible: please try to help wrap up open pull requests and close out open issues.

**Comment by adonath** [comment]
I just opened #1166, which I'd still like to include in gammapy 0.7.

**Comment by woodmd** [comment]
I still need to finish migrating fermipy to use `gammapy.maps` which will probably entail another PR with changes needed for the migration.  I'd say it's not critical to have in 0.7 as long as we can have a 0.8 release on a short timescale (few weeks to month).

**Comment by cdeil** [comment]
I'd like to cut the Gammapy 0.7 release now.

@robertazanin @registerrier @bkhelifi @adonath or anyone interested:

please review http://docs.gammapy.org/dev/changelog.html#gammapy-0p7-release and try out this
http://docs.gammapy.org/dev/tutorials.html#set-up and leave a comment if you think we're ready to go, or if there's something that should be done before cutting the release.

**Comment by cdeil** [comment]
@woodmd - I've made the Gammapy 0.7 release:

https://pypi.python.org/pypi/gammapy/0.7

I'll try to trigger the conda-forge package build tomorrow morning.

**Comment by cdeil** [comment]
Tag is there: https://github.com/gammapy/gammapy/releases/tag/v0.7

**Comment by cdeil** [comment]
Pull request for conda-forge packages: https://github.com/conda-forge/gammapy-feedstock/pull/1

**Comment by cdeil** [comment]
Docs are here: http://docs.gammapy.org/0.7/

**Comment by cdeil** [comment]
Pull request for Macports package update: https://github.com/macports/macports-ports/pull/1361

**Comment by cdeil** [comment]
I've finally sent the release announcement, see here:

* https://groups.google.com/forum/#!topic/gammapy/F0_DsQ1N8Qo
* https://lists.nasa.gov/pipermail/open-gamma-ray-astro/2018-March/000026.html

I've also sent it to those two lists:
* `hess-analysis`
* `cta-wg-as`

Closing this issue and milestone now.

Towards Gammapy 0.8 ...
